design-system/packages/segmented-control/src/SegmentedControl.tsx

```diff
@@ -100,19 +105,18 @@ export const SegmentedControl = ({
       });
     }
   }, [animate, selectedIndex]);
+  const noneSelected = selectedIndex === undefined;
+
+  const getDisabled = (disabled?: boolean, readOnly?: boolean, selected?: boolean) => {
```

Comment:
These parameters are unused, no? And there's a duplicate of this in the Item component?